hpi-schul-cloud/schulcloud-server
# BC-5268 move alert api to nest

# Description

QA note:
Change  tested on  env: 
https://bc-5268-move-alert-to-nest.dbc.dbildungscloud.dev/
https://bc-5268-move-alert-to-nest.nbc.dbildungscloud.dev/
https://bc-5268-move-alert-to-nest.brb.dbildungscloud.dev/
No issues found. Tested for all users, everything works as on the main.
See ticket: https://ticketsystem.dbildungscloud.de/browse/BC-5268


## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-5268
https://github.com/hpi-schul-cloud/nuxt-client/pull/3205
https://github.com/hpi-schul-cloud/schulcloud-client/pull/3442


## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
